sdk/boms/azure-spring-boot-bom/pom.xml

```diff
@@ -57,37 +50,44 @@
 
   <dependencyManagement>
     <dependencies>
+      <dependency>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-sdk-bom</artifactId>
+        <version>1.0.4</version>
```

Comment:
why don't we have a property entry for this version?

Reply:
Use number here is more straightforward. Just like azure-sdk-bom did.